docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do users install and run this?
+
+What is the overall engagement of the Containers experiment?
+**This is the standard Daily Active User (DAU) and Monthly Active User (MAU) analysis.**
+
+This captures data from the users who have the add-on installed, regardless of
+whether they are actively interacting with it.
+
+![](images/kpi-1.png)
+
+### Immediate Questions
```

Comment:
As I understand it, no. @lorchard: our experiment's Telemetry pings will go thru the Test Pilot broadcast channel, so our payloads will be wrapped in the same client ID as the user's other telemetry pings, right? In addition, I believe all Telemetry pings include which add-on IDs are installed? So we could compare search counts, active hours, and page loads of our segment of users with this add-on installed vs. all other users.